trainingMaquetado/index.html

```diff
@@ -31,6 +31,9 @@ <h3 class="title">No internet needed</h3>
       </div>
     </div>
     <div class="lower-container">
+      <h4 class="info">Get info on new updates</h4>
+      <input class="insert" type="text" name="fname"><br>
```

Comment:
since this is a subscription form the button and the input should be inside a `form` tag. I guess this is an email field for the subscription so you should add a `type="email"` and a `name="email"`